Gruntfile.js

```diff
@@ -923,25 +930,21 @@ module.exports = function(grunt) {
     'exec:reset-test-databases',
     'build-node-modules',
     'build-ddoc',
+    'couch-compile:primary',
     'couch-push:test',
     'protractor:performance-tests-and-services',
   ]);
 
-  grunt.registerTask(
-    'unit-continuous',
-    'Lint, karma unit tests running on a loop',
-    ['exec:eslint', 'karma:unit-continuous']
-  );
+  grunt.registerTask('unit-continuous', 'Lint, karma unit tests running on a loop', [
+    'exec:eslint',
```

Comment:
why is eslint in this unit testing task?

Reply:
You're right, it doesn't make sense because eslint won't run continuously. I'll drop the eslint bit - it can be executed explicitly when needed.